ui/src/sandbox.rs

```diff
@@ -190,19 +190,23 @@ impl Sandbox {
 
         let mut cmd = Command::new("docker");
 
-        cmd
-            .arg("run")
+        cmd.arg("run")
             .arg("--rm")
-            .arg("--volume").arg(&mount_input_file)
-            .arg("--volume").arg(&mount_output_dir)
+            .arg("--volume")
+            .arg(&mount_input_file)
+            .arg("--volume")
+            .arg(&mount_output_dir)
```

Comment:
Will do ! Sorry for that, it was a rustfmt slip-up. Had it enabled on file-save, which mangled the entire file. When I noticed that, I copied the entire function to keep my changes after checking the file back out. Lesson learned: Watch these diffs :) !